test/test_common/utility.h

```diff
@@ -430,7 +430,7 @@ class TestUtility {
   static std::string convertTime(const std::string& input, const std::string& input_format,
                                  const std::string& output_format);
 
-  static constexpr std::chrono::milliseconds DefaultTimeout = std::chrono::milliseconds(10000);
+  static constexpr std::chrono::milliseconds DefaultTimeout = std::chrono::milliseconds(30000);
```

Comment:
Does this increase the timeout for all tests?

Reply:
Yes.  Although if the test succeeds, it will not change the time